torch_xla/csrc/ops/topk.cpp

```diff
@@ -18,6 +18,23 @@ xla::Shape NodeOutputShape(const torch::lazy::Value& input, int64_t k,
   return InferOutputShape({GetXlaShape(input)}, lower_for_shape_fn);
 }
 
+xla::Shape NodeOutputShapeSymInt(const torch::lazy::Value& input,
+                                 int64_t k_upper_bound, int64_t dim,
```

Comment:
Can we create a topk_symint.h/cpp instead? it is easier to have a op->file name mapping